test/bidding-strategies/queuepending_test.js

```diff
@@ -0,0 +1,134 @@
+'use strict';
+
+const {QueuePending} = require('../../lib/bidding-strategies/queuepending');
+const {testBiddingStrategy, createMockBid} = require('./bidding-strategies_test.js');
+const sinon = require('sinon');
+const assume = require('assume');
+
+suite('QueuePending Bidding Strategy', () => {
+  let subject = new QueuePending({
+    rootUrl: 'https://fake.example.com',
```

Comment:
Let's use the testing rootUrl for consistency.

Reply:
Can you remind me where we store that property? @djmitche

Reply:
`testRootUrl()` in `taskcluster-lib-urls`.